src/css/base.scss

```diff
@@ -1,15 +1,52 @@
+@import "variables";
```

Comment:
You can make all your imports one line: @import "normalize", "media-queries", "header" .... 
You just need to separate them with a comma and the final impost *needs* a semi colon at the end.